.github/workflows/ci.yml

```diff
@@ -112,21 +112,32 @@ jobs:
           ./vendor/bin/simple-phpunit install
           echo ::endgroup::
 
+      - name: Modify composer.json
+        run: |
+          CURRENT_DIR=$(pwd)
+          for COMPONENT in $(find src/Service -maxdepth 2 -type f -name phpunit.xml.dist -printf '%h\n' | sort)
+          do
+            echo ::group::$COMPONENT
+            echo "$CURRENT_DIR/$COMPONENT"
+            cd "$CURRENT_DIR/$COMPONENT"
+            sed -i -re 's/"async-aws\/core": "[^"]+",/"async-aws\/core": "*@dev",/' composer.json
+            sed -i -re 's/"require": \{/"repositories": [{"type": "path","url": "..\/..\/Core"}],"require": \{/' composer.json
```

Comment:
That breaks the purpose of `prefer-lowest` don't you think?

Reply:
Yes. The code have been updated.